tealeaf/resources/extra

```diff
@@ -0,0 +1 @@
+echo "extra script"
```

Comment:
why is this needed ?

Reply:
For adding script in build phases in xcode project.

Reply:
this shouldn't be needed anymore right ?

Reply:
We need this script to replace.

Reply:
we are adding it from crashlytics. so this is unnecessarry